docs/user_guide/source/engines/bp5.rst

```diff
@@ -156,6 +164,9 @@ This engine allows the user to fine tune the buffering operations through the fo
  StatsLevel                     integer, 0 or 1       **1**, 0
  MaxOpenFilesAtOnce             integer >= 0          **UINT_MAX**, 1024, 1
  Threads                        integer >= 0          **0**, 1, 32
+ FlattenSteps                   boolean               **off**, on, true, false
+ IgnoreFlattenSteps             boolean               **off**, on, true, false
+ Threads                        integer >= 0          **0**, 1, 32
```

Comment:
The `Threads` line doesn't seem connected to the rest

Reply:
Whoops, yes, that line somehow got duplicated.  Will push an update.